src/mpi/init/mpir_init.c

```diff
@@ -87,29 +93,37 @@ int MPIR_Init_impl(int *argc, char ***argv)
     }
 
     int provided;
-    mpi_errno = MPIR_Init_thread_impl(argc, argv, threadLevel, &provided);
+    mpi_errno = MPII_Init_thread(argc, argv, threadLevel, &provided, NULL);
 
     return mpi_errno;
 }
 
-int MPII_Init_thread(int *argc, char ***argv, int user_required, int *provided,
-                     MPIR_Session ** p_session_ptr)
+int MPIR_Init_thread_impl(int *argc, char ***argv, int user_required, int *provided)
 {
-    return MPI_SUCCESS;
+    return MPII_Init_thread(argc, argv, user_required, provided, NULL);
 }
 
-int MPIR_Init_thread_impl(int *argc, char ***argv, int user_required, int *provided)
+int MPII_Init_thread(int *argc, char ***argv, int user_required, int *provided,
+                     MPIR_Session ** p_session_ptr)
 {
```

Comment:
It is not renaming. It is moving the bulk of the code from `MPIR_Init_thread` into `MPII_Init_thread` because the latter is the core function that both `MPIR_Init_thread_impl` and `MPIR_Session_init` calls. Since the bulk of the code was moved, the diff appears as if I swapped the function names.

Reply:
Maybe we can separate into two commits: (1) move code from  `MPIR_Init_thread ` to  `MPII_Init_thread`, no code logic change, (2) change code logic in `MPII_Init_thread` to support session. Will it work?

Reply:
done